hackclub.com.yaml

```diff
@@ -274,6 +274,7 @@ _vercel:
     - vc-domain-verify=ihs.hackclub.com,21e13c0c6af87766f6f5
     - vc-domain-verify=poolesville.hackclub.com,d8099895eaea670eec60
     - vc-domain-verify=wonderland.hackclub.com,0c2e35e271cd0c23752d
+    - vc-domain-verify=northpark.hackclub.com,366fafcc48a7709aeea8
```

Comment:
If you'd like this to point to vercel, you need to create another record for `northpark` in the appropriate spot in the file. Take a look at the README and create a `northpark` record pointing to `cname.vercel-dns.com.`